docs/transactions/iti41.md

```diff
@@ -1,35 +1,228 @@
+* Medication cannot be changed.
+* Dosage instructions cannot be changed.
+* Substitution cannot be disallowed if the existing dispense includes one.
+* Observation code `REFUSE` cannot be issued.
+#####  PADV `CANCEL`
+* Not allowed against DIS or other PADV documents.
+
+#### DIS
+* If [lot number](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-medication.html) (`batch`) is specified in referenced [MTP(`treatmentPlan`) or PRE (`prescription`)](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-medicationdispense.html), then it should match the one in the DIS.
+##### DIS against PRE or MTP:
+* Medication cannot be different if substitution is not allowed by PRE. _This rule might be extended in the future_. 
+* Dispense cannot occur after the end of validity of the PRE document. _This rule might be extended in the future_.
+* Dispense cannot occur after the end of the treatment if specified in the prescription item. _This rule might be extended in the future_.
+* If referenced PRE is not provisional, it should be [dispensable](#FIXME).
+* List of active [ingredients](https://build.fhir.org/ig/hl7ch/ch-emed//StructureDefinition-ch-emed-medication-medicationdispense-definitions.html#Medication.ingredient) should be the same than the one in the referenced [PRE](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-medicationdispense.html) (`prescription`). 
+
+
+### Roles
+* [Patients](http://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html#2.16.756.5.30.1.127.3.10.6-PAT) and their [representatives](http://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html#2.16.756.5.30.1.127.3.10.6-REP) are allowed to publish only the following type of docs:
+    * MTP
+    * PADV with a `COMMENT` observation code with no restriction.
+    * PADV with any other observation code only when referencing a document published by the same patient or a representative of the same patient.     
+* [Healthcare professionals](http://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html#2.16.756.5.30.1.127.3.10.6-HCP) are allowed to publish only the following types of docs:
+    * MTP
+    * PRE
+    * DIS
+    * PADV		
+
+
+### Metadata values check
+Some values from the DocumentEntry are checked against the document (values in metadata and document must be equal):
 
 | DocumentEntry | CH-EMED-EPR                                                  |
 | ------------- | ------------------------------------------------------------ |
 | author        | `Composition.author`                                         |
+| classCode     | `FIXME`                                           |
```

Comment:
for this one as well as formatCode and typeCode, your doc says : "The classCode, formatCode and typeCode attributes must match the document content." - I couldn't find the equivalent in the IG. Where does it occur in the documents?

Reply:
Well it refers to the content in general, that means that if the class format and type are for an MTP document, then the FHIR content must be an MTP document bundle. Then the DocumentEntry.typeCode must match the Composition.type precisely.